mandatory/4-tax.js

```diff
@@ -1,11 +1,18 @@
-/*
+/* 
   SALES TAX
   =========
   A business requires a program that calculates how much sales tax to charge
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+// This function aims to calculate the tax amount to charge on any given product depending on its price and the 20% given sales tax
+function calculateSalesTax(priceOfProduct) {
```

Comment:
It looks like this file contains an error and the test cases failed. Would you be able to fix this?

Reply:
there was an error on the tests, which were raised with CYF and they have now been amended :-)